server.py

```
@@ -553,56 +542,49 @@ async def spinner_task_func():
 
             # --- Tag derivation utility ---
             def derive_tag(tool_calls, audio_files):
```

Pylint (3.3.7)

[refactor] Too many branches (14/12)

(R0912)